### PR TITLE
chore(docker): update Dockerfile & add image-building script

### PR DESCRIPTION
**Pull request by coderzc**
subtask of #840

### CODE REVIEW

File: DockerfileWithHubble [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# 2. Init HugeGraph Sever
+RUN set -e \
+    && cd /hugegraph/hugegraph-server/ \
+    && sed -i "s/^restserver.url.*$/restserver.url=http:\/\/0.0.0.0:8080/g" ./conf/rest-server.properties \
+    && ./bin/init-store.sh
+
+# 3. Init HugeGraph Hubble
+RUN set -e \
+    && cd /hugegraph/hugegraph-server/hugegraph-hubble \
```
Comment (imbajin) [inline DockerfileWithHubble:45]: where is the hubble? it's in toolchain repo now

Comment (coderzc) [reply DockerfileWithHubble:45]: see: https://github.com/apache/incubator-hugegraph/pull/2046/files#diff-7987535fc6eb90c084a6031a104dc578fc6c9d282fb18d8ce5b034f41b737f76R32-R34

Comment (imbajin) [reply DockerfileWithHubble:45]: > see: [#2046 (files)](https://github.com/apache/incubator-hugegraph/pull/2046/files#diff-7987535fc6eb90c084a6031a104dc578fc6c9d282fb18d8ce5b034f41b737f76R32-R34)

who call it? (or just leave for future hubble download URI?)

Comment (coderzc) [reply DockerfileWithHubble:45]: > > see: [#2046 (files)](https://github.com/apache/incubator-hugegraph/pull/2046/files#diff-7987535fc6eb90c084a6031a104dc578fc6c9d282fb18d8ce5b034f41b737f76R32-R34)
> 
> who call it? (or just leave for future hubble download URI?)

just leave for future hubble download URI

File: hugegraph-dist/src/assembly/travis/build-image.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+PROJECT_POM_PATH="${PROJECT_PATH}/pom.xml"
+
+mvn -f "${PROJECT_POM_PATH}" clean package -DskipTests
+
+CONTEXT_PATH=$(mvn -f "${PROJECT_POM_PATH}" -q -N \
+    org.codehaus.mojo:exec-maven-plugin:1.3.1:exec \
+    -Dexec.executable='echo' -Dexec.args='${final.name}')
+    CONTEXT_PATH="${PROJECT_PATH}/${CONTEXT_PATH}"
+
+if [ -n "$2" ]; then
```
Comment (javeme) [inline hugegraph-dist/src/assembly/travis/build-image.sh:31]: can we define a var for $2 for more readable

**Review by Linary — APPROVED** [review]

**Comment by javeme** [comment]
@coderzc ready for review?

File: hugegraph-dist/src/assembly/travis/build-image.sh (hunk 2 above)
Comment (javeme) [inline hugegraph-dist/src/assembly/travis/build-image.sh:31]: can we define a var WITH_HUBBLE=$2 for more readable

**Pull request closed without merging**